src/modules/sign/components/PayloadDetailsCard.tsx

```diff
@@ -175,58 +173,16 @@ const ExtrinsicPart = withRegistriesStore<ExtrinsicPartProps>(
 			}
 		};
 
-		type ArgsList = Array<[string, any]>;
-		type MethodCall = [string, ArgsList];
-		type FormattedArgs = Array<MethodCall>;
-
 		const renderMethodDetails = (): React.ReactNode => {
-			if (formattedCallArgs) {
-				const formattedArgs: FormattedArgs = Object.entries(formattedCallArgs);
-
-				// HACK: if there's a sudo method just put it to the front. Better way would be to order by depth but currently this is only relevant for a single extrinsic, so seems like overkill.
-				for (let i = 1; i < formattedArgs.length; i++) {
-					if (formattedArgs[i][0].includes('sudo')) {
-						const tmp = formattedArgs[i];
-						formattedArgs.splice(i, 1);
-						formattedArgs.unshift(tmp);
-						break;
-					}
-				}
-
-				return formattedArgs.map((entry, index) => {
-					const sectionMethod = entry[0];
-					const paramArgs: Array<[any, any]> = entry[1];
-
-					return (
-						<View key={index} style={styles.callDetails}>
-							<Text style={styles.secondaryText}>
-								Call <Text style={styles.titleText}>{sectionMethod}</Text> with
-								the following arguments:
-							</Text>
-							{paramArgs ? (
-								paramArgs.map(([param, arg]) => (
-									<View key={param} style={styles.callDetails}>
-										<Text style={styles.titleText}>
-											{' { '}
-											{param}:{' '}
-											{arg && arg.length > 50
-												? shortString(arg)
```

Comment:
I spend some time achieve the same on Stylo, although to be able to show the whole decoded payload, all I needed was to ditch this `shortString`, which was effectively shortening valuable information.